src/libraries/Microsoft.PowerFx.Connectors/Execution/HttpFunctionInvoker.cs

```diff
@@ -63,50 +59,97 @@ public HttpRequestMessage BuildRequest(FormulaValue[] args)
             // Header names are not case sensitive.
             // From RFC 2616 - "Hypertext Transfer Protocol -- HTTP/1.1", Section 4.2, "Message Headers"
             var headers = new Dictionary<string, string>(StringComparer.OrdinalIgnoreCase);
+            string body = null;
```

Comment:
>string [](http://example.com/codeflow?start=12&length=6)

This is fine now, but eventually we'll have non-string bodies.

Reply:
even if it's an image, it will be a string at the end
here, body variable stores the ready-for-network string as we call StringContent